horton/scripts/wpart.py

```diff
@@ -119,7 +119,8 @@ def wpart_slow_analysis(wpart, mol):
     operators = []
     for ipure in xrange(npure):
         for iatom in xrange(wpart.natom):
-            operators.append(wpart.cache.load('overlap_operators', iatom)['olp_%05i' % ipure])
+            operators.append(wpart.cache.load('overlap_operators', iatom)[
+                             'olp_%05i' % ipure])
 
```

Comment:
Yeah, the line wrapping is sometimes a bit wonky. But I guess pylint will catch it if it breaks things. The weird thing is that pylint was still complaining after I justified it correctly (according to PEP anyways).

I've unrolled it back to one line now!